### PR TITLE
Stylistic code changes

### PR DESCRIPTION
**Pull request by mattcen**
- Consolidate 'set' options
- Replace basename/dirname with parameter expansion
- Remove function keyword. http://mywiki.wooledge.org/BashGuide/CompoundCommands#Functions
- Consolidate printf command
- Remove unnecessary quotes and braces around variables
- Remove unused variables
- Remove echo -n (Yeah, I know this is bash and it's guaranteed to work; it's still yuck)
- Single-quote printf format string. Shouldn't contain vars, so ensure they don't get expanded even if it does.

### CODE REVIEW

File: ffiler [hunk 1]
```diff
@@ -102,53 +99,53 @@ shift $((OPTIND-1))
 # bomb unless the user has told us how to sort the file(s)
 [[ -z "$fileby" ]] && { bomb "Undefined sort method"; }
 
-for X in $@ ; do
+for X in "$@"; do
   fname="${1:-}"
   shift
 
   # validate the given user input
-  [[ -z "${fname}" ]]   && { usage; exit -1; }
```
Comment (fukawi2) [inline ffiler:110]: If the filename has a space, not quoting this results in bash error "syntax error in conditional expression"

Comment (mattcen) [inline ffiler:110]: Really? What version of bash are you running?

```
mattcen@sonar:tmp$ fname="foo bar"; [[ -z $fname ]] || echo SUCCESS
SUCCESS           
mattcen@sonar:tmp$ 
```

Comment (fukawi2) [inline ffiler:110]: Hmm, unable to replicate. Nevermind :-/

**Comment by fukawi2** [comment]
Changes merged manually in commit 41ce5b7199cdb5

**Pull request closed without merging**